cmd/podman/containers/runlabel.go

```diff
@@ -95,5 +97,8 @@ func runlabel(cmd *cobra.Command, args []string) error {
 			return err
 		}
 	}
+	if rootOptions := utils.RootOptions(cmd); len(rootOptions) > 0 {
+		runlabelOptions.GlobalOptions = strings.Join(rootOptions, " ")
```

Comment:
This will fail if some of the flag values contain spaces. You have to make sure they are quoted correctly.

Reply:
Good thought.  Needs to happen in `RootOptions` though.  An annoying observation is that `.String()` turn `"foo bar"` into `"[foo bar]"`, so I need to continue massaging a bit.

Reply:
Correction, it's slice. Need to look closer.

Reply:
This is still not quoted this will break with `--root "/path/with/sp ace"`, just look at the systemd generate code

Reply:
Did you check the new tests?  Spaces are preserved now. I don't see how a `strings.Join()` can break quoting.

Reply:
As long as the arguments are in an array, spaces are fine but if you join them with spaces into a single string now the user of this string can no longer restore the intended arguments. It cannot now which spaces are part of the argument without quoting.